xml/System.ServiceModel/ClientBase`1+ChannelBase`1.xml

```diff
@@ -139,8 +139,9 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets or sets a value indicating whether <see cref="M:System.ServiceModel.IClientChannel.DisplayInitializationUI" /> attempts to call the <see cref="T:System.ServiceModel.Dispatcher.IInteractiveChannelInitializer" /> objects in the <see cref="P:System.ServiceModel.Dispatcher.ClientRuntime.InteractiveChannelInitializers" /> property or throws if that collection is not empty.</summary>
```

Comment:
Won't consider it blocking, but the summary suggests that an exception has been missed in the documentation

Reply:
Thanks. I'll let the area owners take care of this one, because I can't find the source code for this API.
This PR is just porting.

Reply:
@pgovind, I think you might have mis-parsed the sentence. This api is a boolean property named AllowInitializationUI and is the implementation of the interface property `System.ServiceModel.IClientChannel.AllowInitializationUI`. If you set this to true but have not populated `System.ServiceModel.Dispatcher.ClientRuntime.InteractiveChannelInitializers` with objects which implement the interface `System.ServiceModel.Dispatcher.IInteractiveChannelInitializer`, when you call the interface method `System.ServiceModel.IClientChannel.DisplayInitializationUI()` it will throw. The property itself does not throw.  
Or did you mean the information about throwing an exception is missing from the DisplayInitializationUI documentation?